source/wp-content/themes/wporg-showcase-2022/inc/shortcodes.php

```diff
@@ -35,3 +35,13 @@ function() {
 		return str_replace( 'www.', '', parse_url( $values[0], PHP_URL_HOST ) );
 	}
 );
+
+/**
+ * Returns the number of published posts.
+ */
+add_shortcode(
+	'sites_in_archive_count',
+	function() {
+		return wp_count_posts()->publish;
```

Comment:
Pedantically, this should use `_n()` for handling plurals. That's a bit messy to fit with the shortcode method though - you'd probably wind up with the translation string in the shortcode itself. If the count will always be many then it's not worth the bother (though someone will quibble about correctness, there are edge cases for some languages that have varying singular/plural forms for high numbers). I'd consider that not a blocker.

You could use `number_format_i18n()` on the return value.